xugrid/regrid/regridder.py

```diff
@@ -433,20 +449,32 @@ class OverlapRegridder(BaseOverlapRegridder):
     * ``"mode"``
     * ``"median"``
     * ``"max_overlap"``
+    * percentiles 5, 10, 25, 50, 75, 90, 95: as ``"p5"``, ``"p10"``, etc.
 
     Custom aggregation functions are also supported, if they can be compiled by
     Numba. See the User Guide.
 
+    Any percentile method can be created via:
+    ``method = OverlapRegridder.create_percentile_methode(percentile)``
+    See the examples.
+
     Parameters
     ----------
     source: Ugrid2d, UgridDataArray
     target: Ugrid2d, UgridDataArray
     method: str, function, optional
         Default value is ``"mean"``.
+
+    Examples
+    --------
+    Setup a custom percentile method and apply it:
+
+    >>> p33_3 = OverlapRegridder.create_percentile_method(33.3)
+    >>> regridder = OverlapRegridder(source, target, method=p33_3)
```

Comment:
Good that you are adding an example of the percentiles. I think most users are just looking for the simple use case, so for them it is better to start with the simplest example. And perhaps also guide them to what is the most important method to call (``regrid``)?

```suggestion
    Examples
    --------
    Create an OverlapRegridder to regrid with means:
    
    >>> regridder = OverlapRegridder(source_grid, target_grid, method="mean")
    >>> regridder.regrid(source_data)
    
    Setup a custom percentile method and apply it:

    >>> p33_3 = OverlapRegridder.create_percentile_method(33.3)
    >>> regridder = OverlapRegridder(source_grid, target_grid, method=p33_3)
```